docs/products/mysql/howto/create-missing-primary-keys.rst

```diff
@@ -0,0 +1,79 @@
+Create missing primary keys
+===========================
+
+Find here strategies to create missing primary keys in your Aiven for MySQL service. First, you can determine which tables are missing primary keys by running:
+
+.. code::
+
+    SELECT    
+        tab.table_schema AS database_name,
+        tab.table_name AS table_name,
+        tab.table_rows AS table_rows
+    FROM information_schema.tables tab
+    LEFT JOIN information_schema.table_constraints tco          
+        ON (tab.table_schema = tco.table_schema              
+            AND tab.table_name = tco.table_name
+            AND tco.constraint_type = 'PRIMARY KEY')
+    WHERE
+        tab.table_schema NOT IN ('mysql', 'information_schema', 'performance_schema', 'sys')
+        AND tco.constraint_type IS NULL
+        AND tab.table_type = 'BASE TABLE'; 
+
+
+To see the exact table definition for the problematic tables you can do:
+
+.. code::
+
+    SHOW CREATE TABLE database_name.table_name;
+
+If your table already contains a column or set of columns that can be used as primary key or composite key, then using such column(s) is recommended. Here are some examples of tables definitions and how to create the missing primary keys:
+
+Example 1
+
+.. code::
+
+    CREATE TABLE person (
+    social_security_number VARCHAR(30) NOT NULL,
+    first_name TEXT,
+    last_name TEXT
+);
+
+You can create the missing primary key by first adding the primary key:
+
+.. code::
+
+    ALTER TABLE person ADD PRIMARY KEY (social_security_number);
```

Comment:
When altering the table is Mysql validating that the primary key is unique? If so I would state this because people could experience slower  performance

Reply:
I could not find about the performance but I added a link which says that primary keys are unique, would that work?